app/src/main/java/com/example/ithaca_transit_android_v2/Models/Location.kt

```diff
@@ -1,8 +1,13 @@
 package com.example.ithaca_transit_android_v2.models
 
+import com.example.ithaca_transit_android_v2.Models.LocationType
+import com.squareup.moshi.JsonClass
+
 //Location. - Represents either a bus stop or a google place.
+@JsonClass(generateAdapter = true)
 data class Location (
+    val type : LocationType,
     val name: String,
-    val coordinate: Coordinate,
+    val coordinate : Coordinate,
     val detail: String?
-) {}    
\ No newline at end of file
+) {}
```

Comment:
Lets standardize spacing. What is the convention for this?

Reply:
curly braces can be deleted, i believe